### ISSUE TITLE
CI cmake-format tools 0.6.5 vs 0.6.13 produce different output

### ISSUE THREAD

**Issue by maxgolov** [open-telemetry/opentelemetry-cpp#375]
CI uses older versions of the tools from Ubuntu 18.04. However, the more recent versions of the tools in Ubuntu 20.xx produce different formatting output, thus failing the CI.

CI:

```console
# Ubuntu 18.04
clang-format version 8.0.0-3~ubuntu18.04.2 (tags/RELEASE_800/final)
Running cmake-format 0.6.5.
```

Latest Ubuntu:

```console
# Ubuntu 20.04
clang-format version 8.0.1-9 (tags/RELEASE_801/final)
Running cmake-format 0.6.13.
```

This is a bit interesting, cause one using more recent version of the distro would unlikely be willing to downgrade their version, given that the same `install_format_tools.sh` is gonna deploy the more recent versions.

Morale: `tools/format.sh` can't be trusted if you are using latest Ubuntu.

**Comment by maxgolov** [comment]
I don't know how to solve this. To me the output from `cmake-format 0.6.13` looks more neat / clean, but the CI would fail any `CMakeLists.txt` formatted with latest version of the tool. Maybe we should use more recent version of the tool, and auto-commit re-formatted code back automatically without failing CI (at least for the cmake format, not C++ code format).

**Comment by maxgolov** [comment]
I added the fix in my build tools PR: #377  - to upgrade to 0.6.13, plus manually reformatted all CMakeLists. They look prettier when formatted with recent CMake tooling.

**Comment by maxgolov** [comment]
I'm closing this - since we updated to latest `v0.6.13` , there is a lesser chance of running into this issue. Recent Ubuntu 18.xx and 20.xx both have `v0.6.13` available, with `v0.6.13` being the current latest.
